import_export/admin.py

```diff
@@ -600,17 +600,22 @@ def get_export_form(self):
             Use :meth:`~import_export.admin.ExportMixin.get_export_form_class` or set the new
             :attr:`~import_export.admin.ExportMixin.export_form_class` attribute.
         """
-        warnings.warn(
```

Comment:
We should probably actually keep both of these... One to be triggered when devs are referencing the method from custom view/action code, and the other to for when the method is overridden and the default view code picks up on it.